src/EditorFeatures/CSharpTest/Interactive/NavigateTo/InteractiveNavigateToTests.cs

```diff
@@ -18,6 +18,7 @@
 using Roslyn.Test.Utilities;
 using Xunit;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
This is unrelated to this PR. I'm doing all the suppressions for obsolete API in my other PR https://github.com/dotnet/roslyn/pull/23943/files. Once it's merged, this will go away.